tests/spdx/test_license.py

```diff
@@ -12,6 +12,12 @@ def test_classifier_name() -> None:
     )
 
 
+def test_classifier_name_epl_2_0() -> None:
```

Comment:
I don't think this test case makes much sense. I get that it checks your exact issue case, but I don't see why we should check this exact license among others that we support. I am ok with no test case for that and treat it as a simple data update.

Reply:
I am fine with leaving out a testcase for such a trivial test. I just wanted to check myself that my change resulted in the correct license name that is later used for generating the classifier.